src/main/java/com/zzpj/services/impl/BaseServiceImpl.java

```diff
@@ -20,23 +21,30 @@ public BaseServiceImpl(TRepository repository){
     }
 
     @Override
-    public TModel add(TModel model) {
-        return repository.save(model);
+    public void add(TModel model) {
+        repository.save(model);
     }
```

Comment:
nie lepiej byśmy zwracali wrzucony model w takiej sytuacji? Dla "potencjalnego" frontu przydałby się do wyświetlenia chociażby a tutaj mocno się blokujemy.

Reply:
Ale co chciałbyś wyświetlać? To co dodasz?

Reply:
Dokładnie, chodzi mi o sytuację gdzie jakieś okno i dodajesz rekord (nie musi to być koniecznie książka) i chcesz np. wyświetlić informację o tym lub zaktualizować tabelę to zwraca się obiekt lub przynajmniej Id. Chyba nawet w CRUD repository metoda add zwraca dodany obiekt.

Reply:
no dobra, nie będę się tam jakoś przy tym upierać. możemy to zwracać

Reply:
Chodzi po prostu o to, że nie mam żadnej informacji czy operacja się udała :P

Reply:
Gdyby się nie udała to wyrzuciło by HTTP 400 ;) stąd wiadomo